tools/util.py

```diff
@@ -358,6 +358,20 @@ def extract_number(pattern, string):
     return int(matches[0])
 
 
+def extract_latency(pattern, string):
```

Comment:
Please add a comment or rename the function to indicate that the returned value is in milliseconds.